docs/LINUX.md

```diff
@@ -101,25 +32,58 @@ enabled=1
 
-# Update the package
-sudo yum update logdna-agent
-```
-
-The agent will uninstall the previous version and reuse the existing configuration file, by default
-located in `/etc/logdna.conf`.
-
-If you defined a `logdna-agent.conf` configuration file on a different location, and want to use it instead, you
-can specify it on your systemd unit file:
-
-```unit file (systemd)
-[Service]
-Environment="LOGDNA_CONFIG_FILE=/your/path/to/logdna.conf"
-```
-
-"Next, restart the agent:
+# install the package via yum (skip this step if already installed)
+sudo yum -y install logdna-agent
 
-```shell script
-sudo systemctl restart logdna-agent
+# update the package via yum
+sudo yum update logdna-agent
 ```
-
-[ingestion-key]: https://docs.logdna.com/docs/ingestion-key
+#
+## Usage
+The agent uses [systemd](https://systemd.io/) to run as a Linux daemon. The installed package provides a `systemd` unit file for the `logdna-agent` service which is defined to execute a daemon process with the compiled agent binary. The process started in the service is managed by `systemd` and can be interfaced with using `systemctl`.
+
+---
+### _Enabling the agent_
+- Reload the `systemd` unit files to ensure the most recent version of the agent is used & enable the service via `systemctl`:
+
+  ```bash
+  sudo systemctl daemon-reload
+  sudo systemctl enable logdna-agent
+  ``` 
+---
+### _Configuring the agent_
+└ **Note** [for users upgrading from older versions of [logdna-agent-v2](https://github.com/logdna/logdna-agent-v2) or migrating from the legacy [logdna-agent](https://github.com/logdna/logdna-agent)]: 
+  > Many users will likely already have a configuration file `/etc/logdna.conf` from prior setup(s). While the config variable names have been changed / updated between [logdna-agent](https://github.com/logdna/logdna-agent) and [logdna-agent-v2](https://github.com/logdna/logdna-agent-v2), backwards compatibility has been maintained so older `.conf` files should still be valid. If this is applicable to you and there are no changes that need to be made to your config, you may be able to skip this section and try to start the service using your existing configuration
+#
+
+- By default, `/etc/logdna.conf` is the expected path for the agent's configuration file. Make sure to create this file if it doesn't exist since it's required to start the agent.
+
+    ```bash
+    sudo touch /etc/logdna.conf # it can initalized as an empty file, you will be adding to it in the steps below
+    ```
+    This file stores key-value pairs as environment variables which can be injected into the agent at runtime and manage a variety of configuration options. You can see all the available variable options via `logdna-agent --help` or refer to them [here](README.md#options). If you're coming from the legacy agent, take note of the variables which have been changed, updated, and deprecated when compared to [logdna-agent](https://github.com/logdna/logdna-agent).
```

Comment:
In this case, I think we'd want to reference the legacy project for comparison since these docs would imply they're already looking at the `logdna-agent-v2` project.